### PR TITLE
Show entities on area page

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Show directly linked entities to an area on the area overview page.

Show the total count of entities (also those part of a device in that area)) in the area overview.

image

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/areas/ha-config-area-page.ts [hunk 1]
```diff
@@ -58,9 +64,39 @@ class HaConfigAreaPage extends LitElement {
     | AreaRegistryEntry
     | undefined => areas.find((area) => area.area_id === areaId));
 
-  private _devices = memoizeOne(
-    (areaId: string, devices: DeviceRegistryEntry[]): DeviceRegistryEntry[] =>
-      devicesInArea(devices, areaId)
+  private _memerships = memoizeOne(
```
Comment (spacegaier) [inline src/panels/config/areas/ha-config-area-page.ts:67]:
```suggestion
  private _memberships = memoizeOne(
```
I assume that is a typo or is there a hidden "meme" here 🤣.

File: src/panels/config/areas/ha-config-area-page.ts [hunk 2]
```diff
@@ -87,7 +123,11 @@ class HaConfigAreaPage extends LitElement {
       `;
     }
 
-    const devices = this._devices(this.areaId, this.devices);
+    const { devices, entities } = this._memerships(
```
Comment (spacegaier) [inline src/panels/config/areas/ha-config-area-page.ts:126]:
```suggestion
    const { devices, entities } = this._memberships(
```

File: src/panels/config/areas/ha-config-area-page.ts [hunk 3]
```diff
@@ -58,9 +64,39 @@ class HaConfigAreaPage extends LitElement {
...
+          }
+        } else if (devices.has(entity.device_id)) {
+          indirectEntities.push(entity);
+        }
+      }
+
+      return {
+        devices: Array.from(devices.values()),
+        entities,
+        indirectEntities,
```
Comment (bramkragten) [inline src/panels/config/areas/ha-config-area-page.ts:97]: We don't do anything with these?

**Review by bramkragten — APPROVED** [review]

File: src/panels/config/areas/ha-config-area-page.ts (hunk 3 above)
Comment (balloob) [reply src/panels/config/areas/ha-config-area-page.ts:97]: I'm still thinking about what to do with them 🤷 maybe a toggle to include them ? A new card ?

**Pull request merged**